### PR TITLE
RELATED: RAIL-3156 Sync AFM API sorting and displayForm changes

### PR DESCRIPTION
**Pull request by david-kubecka**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/2990/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/4632/ : SUCCESS in 12m 26s

File: libs/api-client-tiger/src/gd-tiger-model/VisualizationObjectModelV2.ts [hunk 1]
```diff
@@ -28,7 +28,7 @@ export namespace VisualizationObjectModelV2 {
 
     export interface IAttribute {
         localIdentifier: Identifier;
-        displayForm: ObjectIdentifier;
+        label: ObjectIdentifier;
```
Comment (no23reason) [inline libs/api-client-tiger/src/gd-tiger-model/VisualizationObjectModelV2.ts:31]: This is breaking. We should either create V3 or take the risk that the data created in the last week will turn unusable.

Comment (david-kubecka) [reply libs/api-client-tiger/src/gd-tiger-model/VisualizationObjectModelV2.ts:31]: This is breaking also on AFM API level (together with the other sorting change). So I think breaking it here doesn't cause additional problems. (For the context we want to do as much as these breaking changes before tiger GA.)

Comment (no23reason) [reply libs/api-client-tiger/src/gd-tiger-model/VisualizationObjectModelV2.ts:31]: Ok, so to confirm we are ok "nuking" the exisitng insights?

**Review by no23reason — APPROVED** [review]
LGTM

**Comment by yenkins** [comment]
### Sonar scan result
 - No new bug found. Congratulation!

More detail, see in https://sonarqube-dev.intgdc.com/dashboard?id=gooddata-ui-sdk-gate-PR1179

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/891/ : FAILURE in 20m 50s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/1157/ : SUCCESS in 21s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/4651/ : SUCCESS in 12m 25s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-sonarqube-scan-zuul-el7/270/ : SUCCESS in 13m 32s (non-voting)

**Pull request merged**